tests/settings/test_runcard.py

```diff
@@ -77,6 +75,18 @@ def test_serialization(self, runcard):
         assert isinstance(new_runcard_dict, dict)
         assert new_runcard_dict == runcard_dict
 
+    def test_device_id_warning(self):
+        """Test that the initialization of a rncard with `device_id` present raises a deprecation warning"""
+        with catch_warnings(record=True) as w:
+            simplefilter("always")
+            Runcard(**copy.deepcopy(GaladrielDeviceID.runcard))
+            assert len(w) == 1
+            assert issubclass(w[0].category, DeprecationWarning)
+            assert (
+                w[0].message.args[0]
```

Comment:
In the last commit, I added here the `.args[0]`, for having a correct comparison between strings 🙌

Reply:
It worked as it was before but if this works aswell, I dont mind the change!

Reply:
It didn't work mate, I didn't change it for nothing :)
![Screenshot 2024-06-11 at 17 07 57](https://github.com/qilimanjaro-tech/qililab/assets/109400222/1f8a6f34-5d49-4ba7-806e-11beb0f19a4a)